docs/one-click-integrations/jira-integration.md

```diff
@@ -1,29 +1,67 @@
 
-1.  Click **Install Jira** and follow the instructions on the Jira UI to install the app.
+1.  Click **Install Jira App** and follow the instructions on the Jira UI to install the app. You must be a Jira administrator to proceed with this step.
 
-    !!! important
-        This step should be performed by a Jira administrator, since Pulse will only have access to Jira resources that the user setting up the integration has access to.
+    ![Installing the Pulse Jira app](images/jira-installing.png)
 
-        You can only integrate one Pulse organization with each Jira instance.
+1.  Confirm that Pulse successfully created the integration and the necessary webhooks on Jira. If there was an error please [contact support](mailto:pulsesupport@codacy.com).
 
-    ![Installing the Pulse Jira app](images/jira-installing.png)
+<!--TODO Add screenshot for successful installation-->
+
+1.  Choose the strategy to detect incidents that best fits your workflows. See the [section below](#incident-detection-strategy) for a detailed description of each option.
+
+<!--TODO Add screenshot for incident strategy-->
+
+1.  Click **Complete setup**.
+
+Your Jira integration is now complete. Pulse will start loading your data for the last 90 days.
 
-1.  Wait until you get a confirmation that Pulse successfully created the integration and the necessary webhooks on Jira.
+<!--TODO Add screenshot for setup ok-->
 
-    If there was an error please [contact support](mailto:pulsesupport@codacy.com).
+## Automatic incident detection strategies {: id="incident-detection-strategy"}
+
+The Pulse Jira integration can detect **incidents** automatically using [issues assigned with the label "Incident"](#jira-incident-label).
+
+You can also choose [not to detect incidents via Jira](#jira-incident-not-detect).
+
+### Use issues assigned with the label "Incident" {: id="jira-incident-label"}
+
+-   <!--TODO Add detection details-->
+
+### Don't detect incidents via Jira {: id="jira-incident-not-detect"}
+
+-   Pulse doesn't detect incidents automatically using Jira events.
+
+    Choose this option if you want to send to Pulse the information about your **incidents** using another Pulse integration - GitHub or Bitbucket integration, [PagerDuty one-click integration](pagerduty-integration.md), [Pulse CLI](../cli/cli.md), or [Ingestion API](https://ingestion.pulse.codacy.com/v1/api-docs) - or if you don't want Pulse to track incidents data.
 
 ## Collected data
+<!--TODO Update this section-->
+
+The table below lists the data that the Jira integration collects from your Jira instance, together with:
 
-The table below lists the data that the Jira integration collects from your Jira issues, together with the metrics that Pulse calculates from the data to display on the dashboards.
+-   The mapping between the data collected from Jira and the [Pulse data model](https://ingestion.pulse.codacy.com/v1/api-docs#tocs_event)
```

Comment:
@afsmeira The current table in this Collected data section doesn't really show the mapping between the data collected from Jira and the Pulse data model (as we have for [GitHub](https://docs.pulse.codacy.com/one-click-integrations/github-integration/#collected-data) and [Bitbucket](https://docs.pulse.codacy.com/one-click-integrations/bitbucket-integration/#collected-data)).

Can you please provide me with that mapping for Jira, including the incidents? 🙏 

I'll then move the Details into a new section and use this table for the mapping.

Reply:
> The current table in this Collected data section doesn't really show the mapping between the data collected from Jira and the Pulse data model

Because prior to the Jira incidents feature there was no such mapping :)

After the feature we will have the following for the issues that are incidents:
- `incident_id` is the issue key (PUL-1811, for example)
- `time_created` the time the issue was created
- `time_resolved` the time the issue last transitioned to a terminal status (in concrete Jira terms, a status of category 3 'Done' -- https://support.atlassian.com/jira-work-management/docs/workflows-and-statuses-for-the-board/)
- `system` is the compoment if it already exists on Pulse as a system (but I think this limitation will probably be mentioned in a previous section).

Reply:
I updated the "Collected data" section according to what we talked about, excluding the detailed mapping and keeping the existing information for Lead and Cycle time. Let me know if it's ok.